src/main/java/com/laytonsmith/core/functions/PlayerManagement.java

```diff
@@ -4538,4 +4538,72 @@ public Version since() {
 			return CHVersion.V3_3_1;
 		}
 	}
+	@api
```

Comment:
Add @seealso({set_pflight.class}), and then add @seealso({set_pflying.class}) to set_pflight. This will make them show up in each other's documentation.